src/screens/DebugLogs.js

```diff
@@ -38,6 +38,11 @@ export default function DebugLogs() {
     }
   };
 
+  const onDisplayLatestLogs = () => {
+    const logs = logReport.getLogs();
+    alert(logs.map(log => JSON.stringify(log)).join("\n"));
```

Comment:
I imagine next step is to use the Share API to let the user download it?

Example using this library https://github.com/react-native-share/react-native-share/issues/411#issuecomment-442220259

Reply:
yes, also, I assume that the download log page (or download button) should appear when user encounters an error. So we need some implementation to catch the error in the app and put this feature in a correct place

Reply:
The DebugLogs screen I wrote already has the logic to export the logs, you just need to pass in the right data instead of the dummy one. As to the missing implementation, it's a matter of modifying the UI of the error screen/modal to show a button that triggers that same behaviour. Don't worry about catching more errors, that's out of the scope of the task